sphinx/themes/basic/static/basic.css_t

```diff
@@ -768,12 +768,18 @@ div.code-block-caption code {
     background-color: transparent;
 }
 
+/* From https://stackoverflow.com/a/34372191 */
```

Comment:
Is this comment needed? The added styles are not special.

Reply:
It is just for clarity of someone is wondering what all this snippet code is needed, but I can remove it

Reply:
Don't mind. We have other `-moz` and `-webkit` styles. I think they're commonly used styles. So no special comment is needed. Thank you for removal.